src/__tests__/list-test.tsx

```diff
@@ -81,6 +81,27 @@ test('Row as a button', async () => {
     expect(spy).toHaveBeenCalled();
 });
 
+test('Row, keeping its listitem role, containing a button reporting link role', async () => {
+    const spy = jest.fn();
+    render(
+        <ThemeContextProvider theme={makeTheme()}>
+            <RowList>
+                <Row title="Title" onPress={spy} touchableRole="link" />
+            </RowList>
+        </ThemeContextProvider>
+    );
+
+    const rowDiv = screen.getByRole('listitem');
+    expect(rowDiv).toBeInTheDocument();
+
+    const button = screen.getByRole('link', {name: 'Title'});
+    expect(button).toBeInTheDocument();
+    expect(rowDiv).toContainElement(button);
```

Comment:
I'd use `within` here
```suggestion
    const button = within(rowDiv).getByRole('link', {name: 'Title'});
    expect(button).toBeInTheDocument();
```